src/database.cc

```diff
@@ -409,6 +425,15 @@ void Database::SetBusyTimeout(Baton* baton) {
     delete baton;
 }
 
+void Database::SetLimit(Baton* b) {
+    std::unique_ptr<LimitBaton> baton(static_cast<LimitBaton*>(b));
```

Comment:
Ah, modern C++ (i.e. from 21st century)! Looks good to me, but I don't see it used elsewhere, so wouldn't be surprised if it gets frowned upon upstream in case they are trying to support some old C++ compilers.

Reply:
Ah, this is actually me matching the latest style upstream, they've started putting this everywhere.